src/lib.rs

```diff
@@ -207,7 +207,7 @@ pub trait MysqlShim<W: Read + Write> {
 }
 
 /// Information about an authenticated user
-#[derive(Debug, Default, Clone, PartialEq)]
+#[derive(Debug, Default, Clone, PartialEq, Eq)]
```

Comment:
I don't think we should implement `Eq` for this. That new Clippy lint is real annoying :sweat_smile: Just allow the Clippy lint for this.

Reply:
got it.